### DISCUSSION TITLE
Docker Build Error When Using org.opencontainers.image.* Labels

### DISCUSSION THREAD

**Discussion by varunsridharan** [community/community#25435] [Actions]
<p>I am using <code>org.opencontainers.image.*</code> labels in my docker file and while building using github actions it kees throwing errors</p>
<p>Actions Log :<br>
</p><aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/varunsridharan/actions-alpine/runs/1286556219?check_suite_focus=true#step:11:61" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/1884287/181091121-6cebcdc0-e666-4901-b1de-1548ff77d176.jpeg" class="thumbnail onebox-avatar" width="400" height="400">

<h3><a href="https://github.com/varunsridharan/actions-alpine/runs/1286556219?check_suite_focus=true#step:11:61" target="_blank" rel="noopener nofollow ugc">varunsridharan/actions-alpine</a></h3>

<p>Customized Version of Alpine OS With Github Actions Toolkit Buitin - varunsridharan/actions-alpine</p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>DockerFile:<br>
</p><aside class="onebox githubcommit">
  <header class="source">
      <a href="https://github.com/varunsridharan/actions-alpine/commit/ed24b1bf0c78b5eae07d083003ef48762f78f304#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557" target="_blank" rel="noopener nofollow ugc">github.com/varunsridharan/actions-alpine</a>
  </header>
  <article class="onebox-body">
    <div class="github-row">
  <div class="github-icon-container" title="Commit">
    <svg width="60" height="60" class="github-icon" viewbox="0 0 14 16" aria-hidden="true"><path d="M10.86 7c-.45-1.72-2-3-3.86-3-1.86 0-3.41 1.28-3.86 3H0v2h3.14c.45 1.72 2 3 3.86 3 1.86 0 3.41-1.28 3.86-3H14V7h-3.14zM7 10.2c-1.22 0-2.2-.98-2.2-2.2 0-1.22.98-2.2 2.2-2.2 1.22 0 2.2.98 2.2 2.2 0 1.22-.98 2.2-2.2 2.2z"></path></svg>
  </div>

  <div class="github-info-container">
    <h4>
      <a href="https://github.com/varunsridharan/actions-alpine/commit/ed24b1bf0c78b5eae07d083003ef48762f78f304" target="_blank" rel="noopener nofollow ugc">Updated</a>
    </h4>

    <div class="github-info">
      <div class="date">
        committed <span class="discourse-local-date" data-format="ll" data-date="2020-10-21" data-time="11:56:00" data-timezone="UTC">11:56AM - 21 Oct 20 UTC</span>
      </div>

      <div class="user">
        <a href="https://github.com/varunsridharan" target="_blank" rel="noopener nofollow ugc">
          <img alt="varunsridharan" src="https://user-images.githubusercontent.com/1884287/181091123-f8dc1bc6-5fbe-4504-9dd3-e00fc5e87957.jpeg" class="onebox-avatar-inline" width="20" height="20">
          varunsridharan
        </a>
        
      </div>

      <div class="lines" title="changed 1 files with 0 additions and 1 deletions">
        <a href="https://github.com/varunsridharan/actions-alpine/commit/ed24b1bf0c78b5eae07d083003ef48762f78f304" target="_blank" rel="noopener nofollow ugc">
          <span class="added">+0</span>
          <span class="removed">-1</span>
        </a>
      </div>
    </div>

  </div>
</div>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by airtower-luna** [comment, marked as answer]
<p>There’s a backslash missing at the end of the first line of the <code>LABEL</code> instruction in your Dockerfile. That makes Docker interpret the second label as another instruction instead of as part of a label list, and because <code>org.opencontainers.image.authors</code> is no valid Dockerfile instruction it fails.</p>
